src/features/wallet/styled/Wallet.styled.tsx

```diff
@@ -26,25 +17,12 @@ export const Wrapper = styled.div`
     }
     span:last-child {
       font-weight: 500;
-      opacity: 0.3;
+      color: #b3b2b2;
```

Comment:
Doesn't have to be addressed in this PR, but I suggest we move away from hardcoded colors in the future and instead set CSS variables for our theme colors and use those.
For example, if we had our theme in CSS variables, this line would become: `color: var(--colors-gray-300)`